lstchain/calib/camera/r0.py

```diff
@@ -401,5 +401,7 @@ def __init__(self, **kwargs):
 
     def calibrate(self, event):
         for telid in event.r0.tels_with_data:
+            event.r1.tel[telid].trigger_type = event.r0.tel[telid].trigger_type
+            event.r1.tel[telid].trigger_time = event.r1.tel[telid].trigger_time
```

Comment:
Shouldn't that be `event.r1.tel[telid].trigger_time = event.r0.tel[telid].trigger_time` ?

Reply:
yes! I just corrected it